### PR TITLE
Add way to set root password by environnement.

### PR DESCRIPTION
**Pull request by darnuria**
Add a condition test to pass by environement a predetermined
root password for setting up.

Related work in documentation: https://framagit.org/framasoft/peertube/documentation/merge_requests/16

### CODE REVIEW

**Comment by ldidry** [comment]
Maybe you could name your env var differently to make its use more obvious, like `PT_ROOT_PASSWORD`? Otherwise, admins may not know if it’s DB password[1] or root account password.

[1] yeah, I know, it’s dumb because that password is in config file but still, it’s better to have obvious env var.

**Comment by rigelk** [comment]
Additionally I would add a note to:
- https://docs.joinpeertube.org/lang/en/docs/configuration.html 
- https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/production.md#administrator

**Comment by darnuria** [comment]
@all: Thanks for your review! ☺

@ldidry I will make a change, `PASSWORD was a "placeholder" name, is really better  `PT_INITIAL_ROOT_PASSWORD`.

@rigelk : Thanks for the pointers to the documentation.

File: support/doc/production.md [hunk 1]
```diff
@@ -204,6 +204,9 @@ logs. You can set another password with:
 $ cd /var/www/peertube/peertube-latest && NODE_CONFIG_DIR=/var/www/peertube/config NODE_ENV=production npm run reset-password -- -u root
 ```
 
+Alternatively you can set the environnement variable `PT_INITIAL_ROOT_PASSWORD`,
+to with your own administrator password, although it must be 6 characters or more.
```
Comment (rigelk) [inline support/doc/production.md]:
```suggestion
to your own administrator password, although it must be 6 characters or more.
```

**Comment by darnuria** [comment]
update: Fixed some typo.

**Comment by Chocobozzz** [comment]
Thanks @darnuria

**Pull request merged**